pyproject.toml

```diff
@@ -42,7 +42,7 @@ sqlalchemy = "~2.0.23"
 sqlalchemy-utils = "~0.41.1"
 uvicorn = "~0.23.2"
 fastapi-pagination = "^0.12.12"
-fastapi-filter = "^1.0.0"
+fastapi-filters = "~0.2.9"
```

Comment:
Ah sucks that we have to change filtering library.. Is it a known problem on their side with a tracked issue ? The new library has less users and activity it seems.

Reply:
I'll check tomorrow when I get back behind the computer :)

Reply:
I ended up doing a PR #122, to seperate at least the new endpoint and this PR that replaces the filtering library. let me know i can take care of the rebase :pray:

Reply:
an alternative would be to init unique_scans_n to 0